packages/DropdownMenu/src/ContentContainer.styles.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const ContentContainerStyles = styled.div`
+  background: ${tokens.color.white};
+  border: 1px solid ${tokens.border.color};
+  border-radius: ${tokens.popover.borderRadius};
+  box-shadow: ${tokens.popover.shadow};
+  margin-left: ${stylers.spacer(2)};
+  max-width: 320px;
+  min-width: 150px;
+  padding: ${tokens.space} 0;
+  text-align: left;
+
+  ${({ isOpen }) => {
+    if (isOpen === false) {
+      return { opacity: 0 };
+    }
+  }}
+`;
```

Comment:
I'm setting the opacity to 0 for the container to stop the transition of opacity on the initial popover content, at the point of closing the confirmation panel. The Paprika Popover has a 150ms opacity transition built in.

Reply:
How about a more terse one-liner:
```
${({ isOpen }) => (!isOpen ? "opacity: 0;" : "")}
```

Reply:
i've used `${({ isOpen }) => (isOpen === false ? "opacity: 0;" : "")}` as isOpen is not a prop on a content container for the items that don't need a confirmation, so it could also be `undefined`

Reply:
Even better!